docs/scenarios.md

```diff
@@ -0,0 +1,55 @@
+### Scenarios
+Kraken currently supports pod and node based scenarios. 
+
+
+#### Pod chaos scenarios
+Following are the components of Kubernetes/OpenShift for which a basic chaos scenario config exists today. Adding a new pod based scenario is as simple as adding a new config under scenarios directory and defining it in the config.
+
+Component                | Description                                                                                        | Working
+------------------------ | ---------------------------------------------------------------------------------------------------| ------------------------- |
+Etcd                     | Kills a single/multiple etcd replicas for the specified number of times in a loop                  | :heavy_check_mark:        |
+Kube ApiServer           | Kills a single/multiple kube-apiserver replicas for the specified number of times in a loop        | :heavy_check_mark:        |
+ApiServer                | Kills a single/multiple apiserver replicas for the specified number of times in a loop             | :heavy_check_mark:        |
+Prometheus               | Kills a single/multiple prometheus replicas for the specified number of times in a loop            | :heavy_check_mark:        |
+OpenShift System Pods    | kills random pods running in the OpenShift system namespaces                                       | :heavy_check_mark:        |
+
+#### Node chaos scenarios
```

Comment:
I realize there is not all that much information between the pod and node scenarios but should we separate the docs? Might be easier to point to each specific type of object scenarios based on what the user wants to set up

Reply:
Makes sense, divided the scenarios into two separate docs.